tailwind.config.js

```diff
@@ -1,59 +1,72 @@
 module.exports = {
-  future: {
-    // removeDeprecatedGapUtilities: true,
-    // purgeLayersByDefault: true,
-    // defaultLineHeights: true,
-    // standardFontWeights: true
-  },
-  purge: [],
+  purge: [
+    './src/**/*.ts',
+    './src/**/*.tsx',
+    './src/**/*.js',
+    './src/**/*.jsx',
+    './src/**/*.html',
+  ],
+  prefix: 'psl-',
   theme: {
+    screens: {
+      sm: '900px',
+      md: '1100px',
+      lg: '1300px',
+      xl: '1440px',
+      '2xl': '1700px',
+    },
+    fontFamily: {
+      body: 'Roboto, Arial, Helvetica, Helvetica Neue, serif',
+    },
     extend: {
       spacing: {
-        '2px': '2px',
-        '3px': '3px',
-        '6px': '6px',
-        '8px': '8px',
-        '13px': '13px',
-        '14px': '14px',
-        '15px': '15px',
-        '16px': '16px',
-        '18px': '18px',
-        '20px': '20px',
-        '26px': '26px',
-        '28px': '28px',
-        '30px': '30px',
-        '35px': '35px',
-        '36px': '36px',
-        '37px': '37px',
-        '40px': '40px',
-        '41px': '41px',
-        '46px': '46px',
-        '50px': '50px',
-        '60px': '60px',
-        '66px': '66px',
-        '68px': '68px',
-        '70px': '70px',
-        '111px': '111px',
-        '112px': '112px',
-        '220px': '220px',
-        '300px': '300px',
-        '335px': '335px',
-        '352px': '352px',
-      },
-      boxShadow: {
-        xs:
-          '0px 0px 1px rgba(10, 22, 70, 0.06), 0px 32px 40px rgba(10, 22, 70, 0.12)',
-        sm:
-          '0px 0px 1px rgba(10, 22, 70, 0.06), 0px 3px 3px rgba(10, 22, 70, 0.1)',
-        md:
-          '0px 0px 1px rgba(10, 22, 70, 0.06), 0px 6px 8px rgba(10, 22, 70, 0.1)',
-        lg:
-          '0px 0px 1px rgba(10, 22, 70, 0.06), 0px 16px 16px rgba(10, 22, 70, 0.1)',
-        xl:
-          '0px 0px 1px rgba(10, 22, 70, 0.06), 0px 32px 40px rgba(10, 22, 70, 0.12)',
+        '3px': '0.188rem',
+        '13px': '0.813rem',
+        '15px': '0.938rem',
+        '18px': '1.125rem',
+        '22px': '1.375rem',
+        '26px': '1.625rem',
+        '30px': '1.875rem',
+        '35px': '2.188rem',
+        '37px': '2.313rem',
+        '41px': '2.563rem',
+        '46px': '2.875rem',
+        '50px': '3.125rem',
+        '60px': '3.75rem',
+        '63px': '3.938rem',
+        '66px': '4.125rem',
+        '68px': '4.25rem',
+        '69px': '4.313rem',
+        '70px': '4.375rem',
+        '82px': '5.125rem',
+        '111px': '6.938rem',
+        '220px': '13.75rem',
+        '300px': '18.75rem',
+        '335px': '20.938rem',
+        '352px': '22rem',
+        30: '7.5rem',
```

Comment:
can we change this format like others? 30 is not make sense, I think it should be `[number]px`